### ISSUE TITLE
bug: Pin disappears when zooming in to the map

### ISSUE THREAD

**Issue by handekaratas96** [acikkaynak/deprem-yardim-frontend#938]
## Bug Definition
1. Pins disappear. Even though with max zoom in, again with zoom-out > zoom-in .. it disappears again,
2. When there is more than one pin, no action is taken in the details.

** discord username: @Hande Karataş#6520**
Person recording the problem: demeoni#7326

## Bug environment
afetharita.com

## Mobile Phone Information
İphone SE, İOS 16 - Safari ..

https://user-images.githubusercontent.com/124877574/218393779-b787abdc-b534-40be-9af5-675c3164f6aa.mov

**Comment by fdemir** [comment]
It is probably happens because of the auto-fetch mechanism.  We could prevent it by blocking the auto-fetch when use has reached max zoom level.

**Comment by selensr** [comment]
@fdemir some clusters don't show the exact number or marker as it claims. For example in this video, I clicked on the cluster that contains 2 markers but it shows only one.

In addition to that, some of the markers throw errors. Do you need this as reported in another issue?


https://user-images.githubusercontent.com/45594532/218434931-07ddad6b-fce2-41fe-9a84-01d1ec9ed0cb.mp4

**Comment by CyhnYldz** [comment]
We can't reproduce the defect yet. test's will run on more devices for better results.

**Comment by Rayberk** [comment]
#875 should fix that

**Comment by absolutezero13** [comment]
should be fixed on #875 , can you test it on rc again @handekaratas96

**Comment by selensr** [comment]
This issue continues in prod and development as well. The marker does not disappear entirely, but it doesn't show the exact number of markers as it says. @absolutezero13 if you say this is not relevant to this issue, I can create a new one 

https://user-images.githubusercontent.com/45594532/218562932-c31d76aa-8a35-43b1-aabf-5a4d5f8ccac9.mp4

**Comment by sercanerhan** [comment]
tested on several areas after #875 couldn't reproduce the same case.
